docs/01-welcome-to-civicactions/training/orientation-meetings.md

```diff
@@ -146,7 +144,7 @@
 * Begin your [CivicActions Security training process](./security-training.md)
     * Read the [CivicActions Security Policy](../../03-policies/security.md)
```

Comment:
@akaroleff @erogray :

This has already been merged, but I have a DRY issue in that ./security-training.md includes

- security awareness with tools
- securing your laptop
- lastpass

And indeed, even the CivicActions Security Policy. We should have this information in one place of the other, but not both.

Reply:
It is a link to highlights of the section nothing new. But IMHO the whole security section needs a revamp - it's not well organized and overwhelms new hires. There is a trello card to clean it up and better organize it.

Reply:
Agreed re: revamp. I have wanted to make changes to the primary security policy (e.g. pulling out sections on passwords now covered by LastPass and refactoring Drupal sections into the nascent Drupal-specific section of engineering at https://github.com/CivicActions/handbook/blob/master/docs/05-engineering/security-compliance.md#as-drupal-developers) but I could use some help. I asked a few times on #docs (IIRC) some time ago and no response. Also, for the Drupal section I could use a Drupal engineer's help. If you can help coordinate, that would be awesome (but I am slammed through at least the first half if not all of this month)

I added a couple new checklist items to the [Security section clean up in handbook](https://trello.com/c/cQiUz9Wc) Trello card and linked to it from my work board so I don't lose track of it.